.ibm/pipelines/utils.sh

```diff
@@ -1123,3 +1127,44 @@ to_lowercase() {
     echo "${1,,}"
   fi
 }
+
+# Helper function to deploy workflows for orchestrator testing
+deploy_orchestrator_workflows() {
+  local namespace=$1
+
+  rm -rf "${DIR}/serverless-workflows"
```

Comment:
to avoid repeat `${DIR}/serverless-workflows` 3 times, you should create some variables and reuse it.
```
readonly WORKFLOW_REPO="https://github.com/rhdh-orchestrator-test/serverless-workflows.git"
readonly WORKFLOW_DIR="${DIR}/serverless-workflows"
readonly WORKFLOW_MANIFESTS="${WORKFLOW_DIR}/workflows/experimentals/user-onboarding/manifests/"
```